src/Uno.UI/Extensions/NSDateExtensions.iOS.cs

```diff
@@ -32,5 +33,20 @@ public static NSDate ToNSDate(this TimeSpan time)
 		{
 			return DateTime.Today.Add(time).ToNSDate();
 		}
+
+		public static TimeSpan ToTimeSpan(this NSDate date, nint secondsFromGMT)
+		{
+			var offset = TimeSpan.FromSeconds(secondsFromGMT);
+			return date.ToTimeSpan().Add(offset);
+		}
+
+		public static NSLocale ToNSLocale(this string clockIdentifier)
```

Comment:
We don't extend primitives like this for a specific scenario.

Reply:
At least this class should be `internal`

Reply:
You can't put this as _resolved_ until you resolve it.